doc/sphinx/index.rst

```diff
@@ -32,13 +32,16 @@ Community
 
 
 .. toctree::
-   :maxdepth: 1
-   :caption: Contents:
-   :glob:
-
-   src/*
-   src/interface/*
-   src/mesh/*
+   :maxdepth: 2
+   :caption: Chapters
+
+   src/chapters/getting_started
```

Comment:
Does this now require adding files to the list here (or an individual chapter's list) for them to show up? I am worried someone might add documentation in a new file and not know that it wont' automatically show up. 

Maybe we could add a CI test that checks that every rst file in the docs is accounted for in a table of contents? We don't have to do this now... it could be for a later PR.

Reply:
Yes, it's required to add it to this list. I can try to hack up a CI action for this.

Reply:
Ok, I've added an action for this. If there are files that aren't referenced, it should return something like this:
```
➜  parthenon git:(BenWibking/reorg-docs) ✗ uv run scripts/check_docs_toc.py
RST files not referenced by toctree starting at index.rst:
 - doc/sphinx/src/README.rst
 - doc/sphinx/src/building.rst
 - doc/sphinx/src/chapters/getting_started.rst
 - doc/sphinx/src/inputs.rst
 - doc/sphinx/src/outputs.rst
 - ```

Reply:
Nice!